### DISCUSSION TITLE
Trouble Hooking Up Circuit

### DISCUSSION THREAD

**Discussion by BladeCJG** [hneemann/Digital#1197] [Q&A]
![Screenshot 2023-09-15 153727](https://github.com/hneemann/Digital/assets/97693240/515be851-bc86-44a3-84f5-188a7a48b555)

I am trying to make the circuit F = AB + CD. 

1. Why am I getting this error? I have tried using the "Power" component under Misc, that did not work, and I have tried to use "Supply" and "Ground" under wires. 
2. Is there a way to output to an oscilloscope type output device? I am trying to see when F is high/low as well as all of the inputs (A,B,C,D) are high/low from the binary counter.

Please take it easy on me I am new to digital design and am trying to get ahead on some labs I have been struggling with :)

**Comment by hneemann** [comment]
The "Power" component is not required. It is only used to check if VCC and GND are connected to a 74xx chip. If not, an error is generated. It serves as a reminder for students in exercises that a supply voltage is also required. :-)
What should be noted, however, is that connecting to a wire is only possible at its endpoints. Placing a component in the middle of a wire does not connect it to the wire.
